tools/harness-thci/OpenThread_WpanCtl.py

```diff
@@ -216,10 +201,7 @@ def __sendCommand(self, cmd):
         logging.info('%s: sendCommand[%s]', self.port, cmd)
         if self.logThreadStatus == self.logStatus['running']:
             self.logThreadStatus = self.logStatus['pauseReq']
-            while (
-                self.logThreadStatus != self.logStatus['paused']
-                and self.logThreadStatus != self.logStatus['stop']
-            ):
+            while self.logThreadStatus != self.logStatus['paused'] and self.logThreadStatus != self.logStatus['stop']:
```

Comment:
nit: `while self.logThreadStatus not in (self.logStatus['paused'], self.logStatus['stop']):` is more consise

Reply:
After using "python3 -m black -l 119 -S ", the format changed